src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -1295,6 +1301,49 @@ export const _handleUpdateSettings: ControllerHandler<
     })
 }
 
+export const _handleUpdateWebhookSettings: ControllerHandler<
+  { formId: string },
+  FormWebhookSettings | ErrorDto,
+  WebhookSettingsUpdateDto & { userEmail: string }
+> = (req, res) => {
+  const { formId } = req.params
+  const { userEmail } = req.body
+  const settingsToPatch = req.body
```

Comment:
qns: still familiarising myself with typescript. settingsToPatch will include the userEmail, am I right? Can we use it to call `AdminFormService.updateFormSettings(retrievedForm, settingsToPatch) ` directly?